cobra/flux_analysis/deletion.py

```diff
@@ -120,25 +89,36 @@ def _multi_deletion(cobra_model, entity, element_lists, method="fba",
     cobra_model : cobra.Model
         The metabolic model to perform deletions in.
 
+    entity : 'gene' or 'reaction'
+        The entity to knockout (`cobra.Gene` or `cobra.Reaction`)
+
     element_lists : list
-        List of lists `cobra.Reaction`s or `cobra.Gene`s (or their IDs)
+        List of iterables `cobra.Reaction`s or `cobra.Gene`s (or their IDs)
         to be deleted.
 
     method: {"fba", "moma"}, optional
         Procedure used to predict the growth rate.
 
+    number_of_processes : int, optional
+        Number of parallel processes to run. Can speed up the computations
+        if number of knockouts to perform is large. If not passed,
+        will be set to the number of CPUs
+
     solver: str, optional
         This must be a QP-capable solver for MOMA. If left unspecified,
         a suitable solver will be automatically chosen.
 
-    zero_cutoff: float, optional
-        When checking to see if a value is 0, this threshold is used.
-
     Returns
     -------
-    dict of dict
-        A sparse representation of all combinations of pairs of reaction
-        deletions without replacements.
+    pandas.DataFrame
+        A sparse representation of all combinations of entities
+        deletions. The columns are ['ids', 'growth', 'status'], where
+        ids : frozenset([str])
+            all the knockouts performed for one case
+        growth : float
+            growth rate of optimized model
+        status : str
+            solver status
```

Comment:
Github does not let me select line 128 but I think the handling for moma and without QP solver does not work since method is not changed (still moma).